nin/backend/compile.js

```diff
@@ -83,12 +110,14 @@ async function res(projectPath) {
 }
 
 async function collect(projectPath, data) {
-  process.stdout.write(chalk.yellow('\nGenerating demo.html'));
+  const projectMetadata = utils.getProjectMetadata(projectPath);
+  const slug = slugifyTitle(projectMetadata.projectSettings.title);
+  process.stdout.write(chalk.yellow('\nGenerating ' + slug + '.html'));
```

Comment:
No need to get the slug separately here, you can just `projectSettings.title` below the destructuring.